openlibrary/core/lending.py

```diff
@@ -293,7 +293,22 @@ def update_availability_schema_to_v2(v1_resp, ocaid):
 
     url = '{}?{}={}'.format(config_ia_availability_api_v2_url, key, ','.join(ids))
     try:
-        response = requests.get(url, timeout=config_http_request_timeout)
+        client_ip = web.ctx.env.get('HTTP_X_FORWARDED_FOR', 'ol-internal')
+        params = {"scope": "printdisabled"}
+        headers = {
+            "x-preferred-client-id": client_ip,
+            "x-application-id": "openlibrary",
+        }
+        if config_ia_ol_metadata_write_s3:
+            access_key = config_ia_ol_metadata_write_s3['s3_key']
+            secret_key = config_ia_ol_metadata_write_s3['s3_secret']
+            headers["authorization"] = f"LOW {access_key}:{secret_key}"
```

Comment:
Avoid putting secrets into temporary variables.
```suggestion
            headers["authorization"] = "LOW {s3_key}:{s3_secret}".format(**config_ia_ol_metadata_write_s3)
```